### PR TITLE
docs: matching github readme with docs home

### PR DESCRIPTION
**Pull request by galenmarchetti**
## Description:
After updating the Github readme with specific and tangible description of value adds, submitting this to update the home page of docs to match

## Is this change user facing?
YES

### CODE REVIEW

**Comment by galenmarchetti** [comment]
supersedes the home.md changes in https://github.com/kurtosis-tech/kurtosis/pull/328

**Review by leeederek — APPROVED** [review]

File: docs/docs/home.md [hunk 1]
```diff
@@ -5,35 +5,37 @@ slug: '/'
...
 
-## Why Kurtosis?
+In Kurtosis, test environments have these properties:
+- Environment-level portability: the entire test environment always runs the same way, regardless of the host machine
+- Composability: environments can be composed and connected together without needing to know the inner details of each setup
+- Parameterizability: environments can be parameterized, so that they're easy to modify for use across different test scenarios
 
-Docker and Kubernetes are each great at serving developers in different parts of the development cycle: Docker for development/testing, Kubernetes for production. However, the separation between the two entails different distributed app definitions, and different tooling. In dev/test, this means Docker Compose and Docker observability tooling. In production, this means Helm definitions and manually-configured observability tools like Istio, Datadog, or Honeycomb.
+## Architecture
```
Comment (leeederek) [inline docs/docs/home.md:20]: Would actually suggest linking out to the formal architecture page here.

File: docs/docs/home.md [hunk 2]
```diff
@@ -5,35 +5,37 @@ slug: '/'
...
-Kurtosis aims at one level of abstraction higher. Developers can define their distributed applications in Kurtosis, and Kurtosis will handle:
+#### Kurtosis has a validator with:
+- Compile-time safety to quickly catch errors in test environment definitions
+- The ability to dry-run test environment definitions to verify what will be run, before running
 
-With Kurtosis, developers can build with local sandbox environments that demonstrate how their code will work when integrated with the rest of the system. In addition, advanced end-to-end testing workflows are available to teams using the manipulation tooling in the Kurtosis engine runtime which allow them to do end-to-end testing like fault-tolerance, regression, and performance tests.
+#### Kurtosis has a runtime to:
+- Run multi-container test environments over Docker or Kubernetes, depending on how you wish to scale
+- Enable debugging and investigation of problems live, as they're happening in your test environment
+- Manage file dependencies to ensure complete portability of test environments across different test runs and backends
```
Comment (leeederek) [inline docs/docs/home.md:35]:
```suggestion

For more details on the underlying technical architecture of Kurtosis, please see our [Kurtosis Architecture](./explanations/architecture.md)

```

**Comment by mieubrisse** [comment]
Deprecated in favor of https://github.com/kurtosis-tech/kurtosis/pull/364

**Pull request closed without merging**